pkgs/development/python-modules/dvclive/default.nix

```
@@ -32,7 +32,7 @@
 
 buildPythonPackage rec {
   pname = "dvclive";
-  version = "3.41.1";
+  version = "3.42.0";
   pyproject = true;
 
   disabled = pythonOlder "3.8";
```

```suggestion
  disabled = pythonOlder "3.9";
```
https://github.com/iterative/dvclive/blob/170be6f4ee46b3007a02498c3d9bd76d5da0c73d/pyproject.toml#L23